IFC 4.3/UT_Alignment_12d_4/UT_Alignment_12d_4.ifc

```diff
@@ -0,0 +1,46 @@
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1 = IFCPROJECT('2LJesmETv6W9jWXnI$q1Aj', #2, 'UT Alignment 12d 4', '', $, $, $, (#12), #7);
+#2 = IFCOWNERHISTORY(#3, #6, $, .ADDED., $, $, $, 1608356292);
+#3 = IFCPERSONANDORGANIZATION(#4, #5, $);
+#4 = IFCPERSON($, $, 'ljg', $, $, $, $, $);
+#5 = IFCORGANIZATION($, '12d Solutions - Ranch', '12d Solutions - Ranch', $, $);
+#6 = IFCAPPLICATION(#5, '15.0 Alpha 408 [DEVELOP] INT Build 18-12-2020', '12d Model', '12d Model');
+#7 = IFCUNITASSIGNMENT((#8, #9, #10, #11));
+#8 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#9 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#10 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#11 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#12 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #14, #18);
+#13 = IFCLOCALPLACEMENT($, #14);
+#14 = IFCAXIS2PLACEMENT3D(#15, #16, #17);
+#15 = IFCCARTESIANPOINT((0., 0., 0.));
+#16 = IFCDIRECTION((0., 0., 1.));
+#17 = IFCDIRECTION((1., 0., 0.));
+#18 = IFCDIRECTION((0., 1., 0.));
+#19 = IFCSITE('2UD7$ZyUP4jwuszIAU9pHU', #2, 'Default Site', 'Description of Default Site', $, #13, $, $, .ELEMENT., $, $, $, $, $);
+#20 = IFCRELAGGREGATES('0ds0hMKLjBdeC9NpxFvBFX', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#19));
+#21 = IFCFACILITY('2wLm2kwXr9PvJY_vMKvvC6', #2, 'SA  with clothoid no VG', 'Description of SA  with clothoid no VG', $, #13, $, $, .ELEMENT.);
+#22 = IFCRELAGGREGATES('3dGRQ8is19OxGu3HAN$2Kr', #2, 'SiteContainer', 'SiteContainer For All Models', #19, (#21));
+#23 = IFCRELCONTAINEDINSPATIALSTRUCTURE('3SjitDIVb18xVX8J0OQhuE', #2, 'Building Container', 'BuildingContainer for Elements', (#24), #21);
+#24 = IFCALIGNMENT('1lDwpGZZ196A9C_7$_fNcZ', $, 'RS01 with clothoid no VG', '12d Alignment String', 'natural clothoid', $, $, $, $);
+#25 = IFCRELAGGREGATES('3E9jkWubnEhfuo6SbSqGVt', #2, '', '', #24, (#26));
+#26 = IFCALIGNMENTHORIZONTAL('0Rwb_gu3v6Zv174iuvrRAg', #2, 'Horizontal Alignment', '12d Horizontal Alignment', $, $, $, 0., (#28, #30, #32, #34, #36));
+#27 = IFCRELNESTS('2WxrrInhr9f9ee7WcPux5u', #2, '', '', #26, (#28, #30, #32, #34, #36));
+#28 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #29, IFCPLANEANGLEMEASURE(0.), 0., 0., IFCPOSITIVELENGTHMEASURE(38.2922833180901), $, .LINE.);
+#29 = IFCCARTESIANPOINT((0., 0.));
+#30 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #31, IFCPLANEANGLEMEASURE(0.), -0., -150., IFCPOSITIVELENGTHMEASURE(20.), $, .CLOTHOID.);
+#31 = IFCCARTESIANPOINT((38.2922833180901, 0.));
+#32 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #33, IFCPLANEANGLEMEASURE(6.66666666671789E-2), 150., 150., IFCPOSITIVELENGTHMEASURE(74.1184532245796), $, .CIRCULARARC.);
+#33 = IFCCARTESIANPOINT((58.2833962580021, 4.44303370933568E-1));
+#34 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #35, IFCPLANEANGLEMEASURE(5.60789688156486E-1), -150., -0., IFCPOSITIVELENGTHMEASURE(30.), $, .CLOTHOID.);
+#35 = IFCCARTESIANPOINT((152.343929366576, 40.6906546571354));
```

Comment:
same here -> 150., 0.

Reply:
#35 = IFCCARTESIANPOINT((152.343929366576, 40.6906546571354)); 
looks wrong - is identical as in line#37

would assume the values as shown in the PNG -> 128.069...  23.085...